dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/job/BaseMessageManager.java

```diff
@@ -42,13 +47,15 @@
 {
     private final MessageManager messageManager;
     private final SchedulingManager schedulingManager;
+    protected final RenderService renderService;
 
     public BaseMessageManager(
             MessageManager messageManager,
-            SchedulingManager schedulingManager )
+            SchedulingManager schedulingManager, RenderService renderService )
```

Comment:
Can you just inject the objectMapper directly instead? unless you have a specific need for renderService